stable/grafana/README.md

```diff
@@ -71,7 +71,7 @@ You have to add --force to your helm upgrade command as the labels of the chart
 | `ingress.annotations`                     | Ingress annotations (values are templated)    | `{}`                                                    |
 | `ingress.labels`                          | Custom labels                                 | `{}`                                                    |
 | `ingress.path`                            | Ingress accepted path                         | `/`                                                     |
-| `ingress.hosts`                           | Ingress accepted hostnames                    | `[]`                                                    |
+| `ingress.hosts`                           | Ingress accepted hostnames                    | `["chart-example.local"]`                                                    |
```

Comment:
Could you change back to `[]`? If not implemented, please keep as-is.

Reply:
Actually I just adapted the default to as is: https://github.com/helm/charts/blob/master/stable/grafana/values.yaml#L139
Because for me it took quite some time to find out where that ingress with chart-example.com came from.